apps/api-gateway/src/verification/dto/request-proof.dto.ts

```diff
@@ -43,6 +43,153 @@ export class ProofRequestAttribute {
     credDefId?: string;
 }
 
+export declare const PredicateType: readonly [">=", ">", "<=", "<"];
+export type PredicateType = (typeof PredicateType)[number];
+
+export class ProofRequestRestriction {
+  @ApiPropertyOptional({example: '7qEncxNHupBrPBBtNB9e1p:2:Employee Visiting Card:0.1.1'})
+  @IsOptional()
+  @IsNotEmpty()
+  @IsString()
+  // eslint-disable-next-line camelcase
+  schema_id?: string;
```

Comment:
Is there any specific reason for using snack_case?

Reply:
Yes, it is actually the exact key used in that format.